src/server/routes/index.js

```diff
@@ -0,0 +1,58 @@
+exports.notFoundVideo = function notFoundVideo (meta) {
+  meta += `<title>Video not found</title>`
+  return meta
+}
+exports.openGraphHead = function openGraphHead (meta, video) {
+  const ipfsHash = video.ipfsHash
+  const thumbName = video.transcodingStatus.data.result.screenshots[0]
+  const thumbnailUrl =
+    'https://gateway.paratii.video/ipfs/' + ipfsHash + '/' + thumbName
+  const embedUrl = `https://portal.paratii.video/embed/${video._id}`
+  const height = `1080`
+  const width = `1920`
+  meta += `<meta property="og:video:url" content="${embedUrl}">`
+  meta += `<meta property="og:video:secure_url" content="${embedUrl}">`
+  meta += `<meta property="og:video:type" content="text/html">`
+  meta += `<meta property="og:video:width" content="${width}" />`
+  meta += `<meta property="og:video:height" content="${height}" />`
+  meta += `<meta property="og:type" content="video.other" />`
+  meta += `<meta property="og:url" content="${embedUrl}" />`
+  meta += `<meta property="og:title" content="${video.title}" />`
+  meta += `<meta property="og:image" content="${thumbnailUrl}" />`
+  meta += `<meta property="og:description" content="${video.description}" />`
+
```

Comment:
I'm pretty sure you can create a separate handlebars file with all of this content so that you can avoid the string concatenation entirely. I think we want to avoid writing any html in strings

Reply:
ok